domain/src/test/java/org/oppia/domain/topic/StoryProgressControllerTest.kt

```diff
@@ -28,13 +43,52 @@ import org.robolectric.annotation.Config
 import javax.inject.Inject
 import javax.inject.Qualifier
 import javax.inject.Singleton
+import kotlin.coroutines.EmptyCoroutineContext
+
+private const val EXPLORATION_ID_1 = "DUMMY_EXPLORATION_ID_1"
+private const val EXPLORATION_ID_2 = "DUMMY_EXPLORATION_ID_2"
+private const val PROFILE_ID_1 = 1
+private const val PROFILE_ID_2 = 2
+private const val STORY_ID_1 = "DUMMY_STORY_ID_1"
+private const val STORY_ID_2 = "DUMMY_STORY_ID_2"
+private const val TOPIC_ID_1 = "TOPIC_STORY_ID_1"
```

Comment:
Please make values consistent, either remove "DUMMY" from all strings or add them in topic ids also.